platform/src/main/java/org/stellar/anchor/platform/config/PropertySep10Config.java

```diff
@@ -26,12 +30,21 @@ public class PropertySep10Config implements Sep10Config, Validator {
   private List<String> clientAttributionAllowList;
   private List<String> omnibusAccountList;
   private boolean requireKnownOmnibusAccount;
+  private AppConfig appConfig;
   private SecretConfig secretConfig;
 
-  public PropertySep10Config(SecretConfig secretConfig) {
+  public PropertySep10Config(AppConfig appConfig, SecretConfig secretConfig) {
+    this.appConfig = appConfig;
     this.secretConfig = secretConfig;
   }
 
+  @PostConstruct
```

Comment:
`@PostContruct` is called after the `PropertySep10Config` bean is created and validated.